go.mod

```diff
@@ -1,20 +1,25 @@
 module github.com/vmware/antrea-operator-for-kubernetes
 
-go 1.13
+go 1.16
 
 require (
 	github.com/containerd/go-cni v1.0.1
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
-	github.com/go-logr/logr v0.1.0
-	github.com/onsi/gomega v1.10.1
-	github.com/openshift/api v0.0.0-20200701144905-de5b010b2b38
-	github.com/openshift/cluster-network-operator v0.0.0-20200820075439-92e466db53cc
-	github.com/openshift/library-go v0.0.0-20200630145007-34ebc8778b33
-	gopkg.in/yaml.v2 v2.3.0
-	k8s.io/api v0.18.6
-	k8s.io/apimachinery v0.18.6
-	k8s.io/client-go v12.0.0+incompatible
-	sigs.k8s.io/controller-runtime v0.6.3
+	github.com/go-logr/logr v0.4.0
+	github.com/onsi/gomega v1.13.0
+	github.com/openshift/api v0.0.0-20211203184245-fb6f933bb8d5
+	github.com/openshift/cluster-network-operator v0.0.0-20220125073742-83efd7f8d510
+	github.com/openshift/library-go v0.0.0-20210708173104-7e7d216ed91c
+	gopkg.in/yaml.v2 v2.4.0
+	k8s.io/api v0.22.1
+	k8s.io/apimachinery v0.22.1
+	k8s.io/client-go v0.22.1
+	sigs.k8s.io/controller-runtime v0.9.0-beta.0
 )
 
-replace k8s.io/client-go => k8s.io/client-go v0.18.6
+replace (
+	github.com/dgrijalva/jwt-go => github.com/golang-jwt/jwt v3.2.1+incompatible
```

Comment:
migrate to `golang-jwt`.
[https://pkg.go.dev/github.com/golang-jwt/jwt#section-readme](https://pkg.go.dev/github.com/golang-jwt/jwt#section-readme)